Source/Scene/ImagerySplitDirection.js

```diff
@@ -0,0 +1,42 @@
+/*global define*/
+define([
+        '../Core/freezeObject'
+    ], function(
+        freezeObject) {
+    'use strict';
+
+    /**
+     * The direction to display an ImageryLayer relative to the {@link Scene#imagerySplitPosition}.
+     *
+     * @exports ImagerySplitDirection
+     *
+     * @see ImageryLayer#splitDirection
+     */
+    var ImagerySplitDirection = {
+        /**
+         * Display the ImageryLayer to the left of the {@link Scene#imagerySplitPosition}.
+         *
+         * @type {Number}
+         * @constant
+         */
+        LEFT : -1.0,
```

Comment:
Are these intended to be floating-point?  If so, all three should end in `.0`.  If these are intended to be integer, none should have `.0`.